nengo_dl/compat.py

```diff
@@ -175,5 +188,14 @@ def is_sparse(sig):
     General = LinearFilter.General
 
     def make_linear_step(synapse, input_shape, output_shape, dt):
-        """Call synapse.make_step to compute everything."""
-        return synapse.make_step(input_shape, output_shape, dt, None)
+        """Call synapse.make_step to compute A/B/C/D."""
+        state = synapse.make_state(input_shape, output_shape, dt)
+        return synapse.make_step(input_shape, output_shape, dt, None, state)
```

Comment:
Maybe ``synapse.make_step(input_shape, output_shape, dt, rng=None, state=state)`` just so it's clear what that `None` is.